subxt/src/storage/storage_address.rs

```diff
@@ -107,100 +127,24 @@ where
         }
     }
 
-    /// Return bytes representing this storage entry.
-    pub fn to_bytes(&self) -> Vec<u8> {
-        let mut bytes = Vec::new();
-        super::utils::write_storage_address_root_bytes(self, &mut bytes);
-        for entry in &self.storage_entry_keys {
-            entry.to_bytes(&mut bytes);
-        }
-        bytes
-    }
-
     /// Return bytes representing the root of this storage entry (ie a hash of
-    /// the pallet and entry name).
-    pub fn to_root_bytes(&self) -> Vec<u8> {
+    /// the pallet and entry name). Use [`crate::storage::StorageClient::address_bytes()`]
+    /// to obtain the bytes representing the entire address.
+    pub fn root_bytes(&self) -> Vec<u8> {
```

Comment:
I like `to_root_bytes` better but no biggie

Reply:
I'm trying to remember why I changed it now!

Reply:
I changed it to to_root_bytes; you're right, it feels nicer!